dist/js/tooltip.min.js

```diff
@@ -1 +1 @@
on(t,e){var i,o=this.$(t)[0],n=this.xrel,s={left:0,top:0,width:0,height:0,box:this.box(o)};if(o){s.left=o.offsetLeft,s.top=o.offsetTop,s.width=o.offsetWidth,s.height=o.offsetHeight;for(var r=o;(r=r.parentNode)&&"BODY"!==r.tagName;)r.scrollTop&&(s.top-=r.scrollTop),r.scrollLeft&&(s.left-=r.scrollLeft);for(;o=o.offsetParent;)i=getComputedStyle(o),n.test(i.getPropertyValue("position"))!==!0?(s.left+=o.offsetLeft,s.top+=o.offsetTop):s.parent=o}return s},frame:function(t){var e=document.documentElement,i=document.body,o=this.$(t)[0],n=o;if(o){for(;o.scrollTop<=0&&o!==i;)o=o.parentNode,n=o;return o===i&&(o.scrollTop<e.scrollTop||o.scrollLeft<e.scrollLeft)&&(n=e),{node:o,top:n.scrollTop,left:n.scrollLeft,scroll:n.scrollTop,width:o.offsetWidth,height:o.offsetHeight}}return{node:null}},position:function(t,e){var i,o=t.attr("data-position")||this.config.position;return"none"!==o&&(i=this._position(o,this.offset(e,!0),this.offset(t),this.frame(e)),i&&(this.each(this.config.map,function(t,i){e.removeClass(i)}),e.addClass(i.key),this.emit("position",t[0],e[0],i),e.css({left:i.left,top:i.top}))),this},modal:function(t){var e,i=this.$(t),o=i.attr("aria-describedby")||"";return o?(e=this.$("#"+o),this.connect(i,e),e):this.link(t).modal(t)},focus:function(t,e){var i=e.find(this.selector("kill"));return i.length<1&&(i=this.html("kill"),i.on("focus.mk",function(){t.focus()}),e.append(i)),this},show:function(t){var e,i=this.$(t);return this.unlocked(i)&&this.isHidden(i)&&(this.hideAll(),e=this.modal(t),this.delay(function(){e.removeClass("out").addClass("in").attr("aria-hidden","false"),this.position(i,e),this._bindModalDown(i,e),this.isFocusable(e)&&this.focus(i,e),i.addClass("active"),this.emit("show",i[0],e[0])}),this.transition(e,function(t,e){e.removeClass("in")})),this},hide:function(t,e){var i,o,n=this.$(t),s=n.attr("data-action");return this.unlocked(n)&&this.isOpen(n)&&(i=this.modal(t),o=e!==!0&&"click"!==s&&this.isFocusable(i)&&this.config.delay||1,this.delay(function(){i.removeClass("in").addClass("out").attr("aria-hidden","true"),this._unbindModalDown(i),e!==!0&&this.transitions||(i.removeClass("out"),this.clearTransitions(i)),n.removeClass("active"),this.emit("hide",n[0],i[0])},o),this.transition(i,function(t,e){e.removeClass("out")})),this},hideAll:function(){var t=this.$(this.selector());return this.each(t,function(t){this.isOpen(t)&&this.hide(t,!0)})},toggle:function(t){return this.isOpen(t)?this.hide(t):this.show(t)},isOpen:function(t){return!this.isHidden(t)},isHidden:function(t){return"false"!==this.modal(t).attr("aria-hidden")},lock:function(t){var e=this.$(t);return e.hasClass(this.name)&&!e.hasClass("locked")&&(e.addClass("locked"),this.emit("lock",e[0],!0)),this},unlock:function(t){var e=this.$(t);return e.hasClass(this.name)&&e.hasClass("locked")&&(e.removeClass("locked"),this.emit("lock",e[0],!1)),this},locked:function(t){return this.$(t).hasClass("locked")},unlocked:function(t){return this.$(t).hasClass("locked")!==!0}}),e.get("Tooltip")});
```

Comment:
Looks like there actually ARE some code differences here. You can see them about half way down the file in the _blur method. Look for "_blur:". That's a little concerning as it makes me think the tooltip src file was changed somewhere (or not change and was stale ie: overwrote a change). I'd test the tooltip minified file somewhere.